carbon-design-system/carbon
# fix(react): update public exports

Closes #11131

This PR addresses an issue that came up from how we merged our public API. It addresses Toggletip exports not being available and removes deprecated components in v11

#### Changelog

**New**

**Changed**

- Update entrypoint to export Toggletip components

**Removed**

- Removed Toolbar and ToggleSmall components that were removed in v11

#### Testing / Reviewing

- Verify the public API of `@carbon/react` contains Toggletip exports
- Verify the public API of `@carbon/react` no longer contains removed components
